recipes/qt/6.x.x/conanfile.py

```diff
@@ -767,30 +772,30 @@ def package(self):
         with self._build_context():
             cmake = self._configure_cmake()
             cmake.install()
-        tools.save(os.path.join(self.package_folder, "bin", "qt.conf"), qt.content_template("..", "res", self.settings.os))
+        save(self, os.path.join(self.package_folder, "bin", "qt.conf"), qt.content_template("..", "res", self.settings.os))
         self.copy("*LICENSE*", src="qt6/", dst="licenses")
         for module in self._get_module_tree:
             if module != "qtbase" and not self.options.get_safe(module):
-                tools.rmdir(os.path.join(self.package_folder, "licenses", module))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+                rmdir(self, os.path.join(self.package_folder, "licenses", module))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         for mask in ["Find*.cmake", "*Config.cmake", "*-config.cmake"]:
-            tools.remove_files_by_mask(self.package_folder, mask)
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la*")
-        tools.remove_files_by_mask(self.package_folder, "*.pdb*")
-        tools.remove_files_by_mask(self.package_folder, "ensure_pro_file.cmake")
+            rm(self, self.package_folder, mask, recursive=True)
+        rm(self, os.path.join(self.package_folder, "lib"), "*.la*", recursive=True)
+        rm(self, self.package_folder, "*.pdb*", recursive=True)
+        rm(self, self.package_folder, "ensure_pro_file.cmake", recursive=True)
```

Comment:
```suggestion
            rm(self, mask, self.package_folder, recursive=True)
        rm(self, "*.la*", os.path.join(self.package_folder, "lib"), recursive=True)
        rm(self, "*.pdb*", self.package_folder, recursive=True)
        rm(self, "ensure_pro_file.cmake", self.package_folder, recursive=True)
```
conan.tools.files.rm has pattern argument before folder argument, cf https://docs.conan.io/en/latest/reference/conanfile/tools/files/basic.html#conan-tools-files-rm

Reply:
That should teach me to not do quick search and replace edits. Thank you